pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -91,6 +92,29 @@ func newStateImpl(checkpointDirectory, checkpointName string) state.State {
 	return stateImpl
 }
 
+// PodResourceSummary save pod resize info
+type PodResourceSummary struct {
+	// TODO: Add pod-level resources here once resizing pod-level resources is supported
+	// Resources v1.ResourceRequirements
+	InitContainers []ContainerAllocation
+	Containers     []ContainerAllocation
```

Comment:
This can simplify `getPodResizeInfo`, I guess.
```suggestion
        InitContainers []ContainerAllocation `json:"InitContainers,omitempty"`
        Containers     []ContainerAllocation `json:"Containers,omitempty"`
```

Reply:
Applied